### DISCUSSION TITLE
More selection logic of (railgun) loader

### DISCUSSION THREAD

**Discussion by Charile1890** [FakeFishGames/Barotrauma#7832] [Ideas and suggestions]
(please refer to my post image)
Before the latest update , if I link railgun to loader in order from 1~3
I can fire it in B mode instead of A mode, that is, always use first loaders unless it is empty.
This design is useful to manage the high value (and dangerous) shell such as nuclear shell or shell including C-4.
I can maintain 1st loader empty , fire 2nd loader with full of normal shell until I dicide to use nuclear shell , than call my crewmate to load it into 1st loader (or do it by myself when single-play).
But it is failed after the latest update (now can only fire in A mode).

Although the issue is due to "an intentional fix to #7324 "
I think these two fire option should can be choiced by the player.

![text](https://user-images.githubusercontent.com/96295564/146554928-ccd6dae1-ab8d-4d5a-865c-6524b14bff98.png)

**Comment by MaaiG** [comment]
* I think that "Loader Base" and "Turret Hardpoint" should be connected by wires (pipes) .
The "Loader Base"  must have an output for the "ammunition counter".
This allows to create switching circuits and etc.

* "Ventilation" should also be connected by pipes (similar to wires) and have switching valves to control the air flow.
